src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -2863,15 +2863,6 @@ private void RemoveDirectoryInfoItem(DirectoryInfo directory, bool recurse, bool
                 continueRemoval = ShouldProcess(directory.FullName, action);
             }
 
-            //if this is a reparse point and force is not specified then warn user but dont remove the directory.
```

Comment:
I've updated the PR description and removed the unused DirectoryReparsePoint message string.